src/MailEclipse.php

```diff
@@ -395,6 +395,10 @@ public static function getMailableTemplateData($mailableName)
     public static function generateMailable($request = null): JsonResponse
     {
         $name = self::generateClassName($request->input('name'));
+        $defaultDirectory = 'Mail';
+
+        $mailableDir = config('maileclipse.mailables_dir');
+        $customPath = substr($mailableDir,strpos($mailableDir,$defaultDirectory) + strlen($defaultDirectory) + 1);
```

Comment:
Hi there @Bhagyrajaj,

Would it be possible to correct this according to StyleCI please:
https://github.styleci.io/analyses/GDQpEO

Reply:
sure

Reply:
Thank you